developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md

```diff
@@ -0,0 +1,161 @@
+---
+title: text2vec-jinaai
+sidebar_position: 13
+# tags: ['text2vec', 'text2vec-jinaai', 'jinaai']
+---
+
+
+## Overview
+
+Weaviate uses the `text2vec-jinaai` module to obtain vectors.
+- [JinaAI Embeddings](https://jina.ai/embeddings/)
+
+Key notes:
+
+- JinaAI requires a third-party API key.
+- To use the [nearText search operator](/developers/weaviate/api/graphql/search-operators.md#neartext), enable the text2vec-jinaai model.
```

Comment:
This should be rephrased because `nearText` can be used with any one of the many vectorizers imo. How about:

- Enabling the text2vec-jinaai model will enable the [nearText search operator](/developers/weaviate/api/graphql/search-operators.md#neartext).

Reply:
Updated